native/templates/ios/AppDelegate.mm

```diff
@@ -25,24 +25,25 @@ of this software and associated engine source code (the "Software"), a limited,
  THE SOFTWARE.
 ****************************************************************************/
 
-#import "platform/ios/AppDelegate.h"
+#import "AppDelegate.h"
 #import "ViewController.h"
-#import "platform/ios/View.h"
+#import "View.h"
 
 #include "platform/ios/IOSPlatform.h"
-#include "platform/ios/service/SDKWrapper.h"
+#import "platform/ios/AppDelegateBridge.h"
+#import "service/SDKWrapper.h"
 
 @implementation AppDelegate
-
-cc::IOSPlatform *_platform = nullptr;
-
 @synthesize window;
+@synthesize appDelegateBridge;
 
 #pragma mark -
 #pragma mark Application lifecycle
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    [[SDKWrapper shared] application:application didFinishLaunchingWithOptions:launchOptions];
+    appDelegateBridge = [[AppDelegateBridge alloc] init];
```

Comment:
Where to destroy the bridge?

Reply:
Because sdkwrapper itself is a static class that contains other users' listening events. When destroying an event, it is also necessary to handle the event before destroying it. I'm not sure where a release would be better, so kept the previous design

Reply:
Got it, should think about it in future.